parcels/particleset/baseparticleset.py

```diff
@@ -332,13 +485,20 @@ def execute(self, pyfunc=AdvectionRK4, pyfunc_inter=None, endtime=None, runtime=
         :param postIterationCallbacks: (Optional) Array of functions that are to be called after each iteration (post-process, non-Kernel)
         :param callbackdt: (Optional, in conjecture with 'postIterationCallbacks) timestep inverval to (latestly) interrupt the running kernel and invoke post-iteration callbacks from 'postIterationCallbacks'
         """
-        # check if pyfunc has changed since last compile. If so, recompile
+        # check if pyfunc has changed since last compile. If so, recompile.
+        # COMMENT #1034: this still needs to check that the ParticleClass name also didn't change!
```

Comment:
no, don't think so - to much work to still include it in here. Will be separate - or perhaps even not needed, if users don't get to the idea to change a particle set after creation and between multiple `execute(...)` calls. If it wasn't done wrong until now, then perhaps it's not an issue. It's just that there exists this possibility and that it would break this execute function ...